activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -78,6 +78,32 @@ def #{method_name}=(value)           # def includes_values=(value)
 
     alias extensions extending_values
 
+    # Allows preloading of associations on a relation or array of records,
+    # in the same way that #preload does. Mutates the +records+ argument.
```

Comment:
Should `preload` be linked here? Or does RDoc do it automatically via `#` and I just don't remember? 😄

Reply:
Other methods seem to have it linked automatically through `#` but I'm not sure!